addons/web/static/tests/fields/relational_fields/field_many2one_tests.js

```diff
@@ -2926,6 +2926,60 @@ QUnit.module('fields', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('search more in many2one: dropdown click', async function (assert) {
+            assert.expect(8);
+
+            for (let i = 0; i < 8; i++) {
+                this.data.partner.records.push({id: 100 + i, display_name: 'test_' + i});
+            }
+
+            // simulate css using matching modal inside the dom (not at first child)
+            const fakeDialog = document.createElement('div');
+            fakeDialog.innerHTML = `<div class="pouet">
+                <div class="modal"></div>
+            </div>`;
+            document.body.append(fakeDialog);
```

Comment:
Well, you could also use jQuery (I know, I know... but it's not going away anytime soon in the tests...)

Reply:
Even if I don't know why it breaks on `.innerHTML` but not `.html()` :thinking: